app/models/hbx_enrollment.rb

```diff
@@ -1366,4 +1374,18 @@ def benefit_group_assignment_valid?(coverage_effective_date)
       false
     end
   end
+
+  def find_relationship(relationships)
+   rel = relationships.uniq.join(",")
+   case rel
+    when 'employee'
+     'employee_only'
+    when 'employee,spouse', 'employee,domestic_partner'
+     'employee_and_spouse'
+    when 'employee,child_under_26'
+     'employee_and_one_or_dependents'
+    else
+     'family'
+    end
+  end
```

Comment:
I think if we add child first before spouse we will get "employee_and_one_or_dependent "  composite tier for that enrollment and if we add spouse(to add spouse we need to do plan shopping again which creates new enrollment. once it gets activated the old enrollment will be cancelled.) then the new enrollment composite tier will be "family". Correct me if i am wrong.